Update Animal Genomics site navigation (#2662)
nf-core/website: sites/main-site/src/content/special-interest-groups/animal-genomics/index.md

# Activities

The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will feature a diverse range of topics in the field of animal genomics, including:

- Inspiring talks from top genomics researchers.
- Novel ongoing efforts in animal genomics (new consortiums, new calls, etc).
- More technical sessions on practical examples of how to use nf-core pipelines applied to animal genomics research.
  The discussion will often be a follow-up on the talk.

# How to join

> Join the [nf-core Slack](/join#slack) and find us in the #animal-genomics channel

Reviewer: ```suggestion
Join the [nf-core Slack](/join#slack) and find us in the [#animal-genomics channel](https://nfcore.slack.com/archives/C06M8VDEHLG)
```

# Activities

> The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.

Reviewer 2: ```suggestion
The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will feature a diverse range of topics in the field of animal genomics, including:
* Inspiring talks from top genomics researchers.
* Novel ongoing efforts in animal genomics (new consortiums, new calls, etc).
* More technical sessions on practical examples of how to use nf-core pipelines applied to animal genomics research. 
The discussion will often be a follow-up on the talk.
```

# How to join

> Join the [nf-core Slack](/join#slack) and find us in the [#animal-genomics channel](https://nfcore.slack.com/archives/CE5EL6326)

Reviewer 2: The link does not take me to the #animal-genomics channel thus, I will delete it
```suggestion
Join the [nf-core Slack](/join#slack) and find us in the #animal-genomics channel
```
